typedoc.json

```diff
@@ -5,7 +5,7 @@
   "plugin": "typedoc-plugin-markdown",
   "readme": "none",
   "gitRevision": "master",
-  "exclude": ["src/**/!(secure|checkpointTrie|baseTrie).ts", "test/**/*.ts"],
+  "exclude": ["src/**/!(secure|checkpointTrie|baseTrie|walkController).ts", "test/**/*.ts"],
```

Comment:
i see you added this exclude but the docs did generate for WalkController (which is good imo since we are exporting it) - did this addition do something meaningful?

Reply:
Yes, I actually want that typedoc also documents `walkController`. Note that this exclude statements excludes everything except secure/checkpoinTrie/baseTrie/walkController.ts, hence the `!` (`NOT`) operator.

Reply:
OH right 😄 my bad